docker-compose-prod.yml

```diff
@@ -4,8 +4,11 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.prod
+      args:
+        VITE_ARGOS_API_BASE_URL: ${VITE_ARGOS_API_BASE_URL}
+        VITE_FILE_MANAGER_API_BASE_URL: ${VITE_FILE_MANAGER_API_BASE_URL}
     ports:
-      - "5175:80"
+      - "5173:80"
```

Comment:
This port is using develop

Reply:
Okay I've returned it to its original value